ipalib/config.py

```diff
@@ -518,6 +518,10 @@ def _finalize_core(self, **defaults):
             self._merge_from_file(self.conf)
             self._merge_from_file(self.conf_default)
 
+        # Set nss_dir to nssdb directory in confdir
+        if 'nss_dir' not in self:
+            self.nss_dir = self._join('confdir', 'nssdb')
```

Comment:
NACK, the default should be whatever is defined in `ipaplatform.paths`, changing the default is out of the scope of this fix.